pacs-integration-webapp/pom.xml

```diff
@@ -107,7 +107,7 @@
         <dependency>
             <groupId>org.bahmni.module</groupId>
             <artifactId>web-clients</artifactId>
-            <version>0.77-SNAPSHOT</version>
+            <version>0.93-SNAPSHOT</version>
```

Comment:
@buvaneswari-arun is it required to have the latest version of web-clients here ?

Reply:
Sync was not working in product QA. Hence moving to the latest web-client. Also in all other modules, the latest web-clients jar is updated to 0.93 - hence for consistency purpose too this I felt would be apt.